src/app/api/profile/route.ts

```diff
@@ -0,0 +1,27 @@
+import { NextRequest, NextResponse } from 'next/server'
+
+import { updateUserProfile } from '@/services/actions/userActions'
```

Comment:
No need for this file. We use server actions instead.

check the services/actions folder and see how we use this.

The main reason is that is strongly typed.